BuildProject/android/app/build.gradle

```diff
@@ -71,6 +71,17 @@ android {
 
         }
     }
+
+    packagingOptions {
```

Comment:
Was there a build failure caused by this? Unfortunately, I'm much less knowledgeable about the Android stuff.

Reply:
Yea.  It looks like Gradle is linking duplicate `.so`s.  I don't know if this was a previous issue or a new version issue.